File: pset6/MarioMore/mario.py
from cs50 import get_int

def main():
    while True:
        height = get_int("Height of pyramid: ")
        if height > 0 and height < 9:
            break
        else:
            print("**** Enter valid height ****")
    pyramid(height)

def pyramid(height):
    for i in range(1,height+1 ):
        print(" "*(height - i ) + "#"*i + "  "+ "#"* i)
    
if __name__ == "__main__":
    main()

File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void){
    int h,i,j,k,k1 ;
    do{
        h = get_int("Height : ");
    }while(h < 1 || h > 8);

    for(i = 0 ; i < h ; i++)
    {
        for(j = 0 ; j < h-i-1 ; j++)
        {
            printf(" ");
        }
        for(k = 0 ; k <= i ; k++)
        {
            printf("#");
        }

        printf("  ") ;

        for(k1 = 0 ; k1 <= i ; k1++)
        {
            printf("#");
        }
       

        printf("\n");
    }
}